src/components/dialogs/input-hooks.js

```diff
@@ -442,7 +445,35 @@ export const useAutocompleteField = ({
         }
     }, [selectedValue]);
 
+    useEffect(() => {
```

Comment:
Misunderstanding this useEffect, values can now be a promise or a function ?
Can we make this code more clear ?

Reply:
Yes, values can now be either an array, a Promise or a function.
Can I make it clearer ? With pure code, no.
With a comment reminding that Promise.resolve given a parameter that is not a Promise creates a Promise that will call `then` with that value. Can not see a better way. What do you propose ?